msf_updates/src/pose_msf/pose_sensormanager.h

```diff
@@ -170,16 +170,16 @@ class PoseSensorManager : public msf_core::MSF_SensorManagerROS<
     P.setZero();  // Error state covariance; if zero, a default initialization in msf_core is used
 
     p_vc = pose_handler_->GetPositionMeasurement();
-    q_cv = pose_handler_->GetAttitudeMeasurement();
+    q_vc = pose_handler_->GetAttitudeMeasurement();
```

Comment:
Sorry about that! For me this was the case, so this notation made it easier to understand for me... Simon, can you please elaborate on your idea?